lab_1/main.py

```diff
@@ -21,6 +29,13 @@ def remove_stop_words(tokens: list, stop_words: list) -> list or None:
     :param stop_words: a list of stop words
     :return: a list of tokens without stop words
     """
+    for i in tokens:
+        if i.isalpha:
+            if i in stop_words:
+                tokens.remove(i)
```

Comment:
what would happen if I wanted to work with these tokens later, in other functions?